x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/components/back_link.tsx

```diff
@@ -17,7 +17,8 @@ interface Props {
 export function BackLink({ queryParams, href: integrationsHref }: Props) {
   const { onboardingLink } = useMemo(() => {
     return {
-      onboardingLink: queryParams.get('observabilityOnboardingLink'),
+      onboardingLink:
+        queryParams.get('observabilityOnboardingLink') || queryParams.get('onboardingLink'),
```

Comment:
SecuritySolution uses onboardingLink url param to get the `back link` (I'm trying to name it more general, open for suggestion):

Back link video here:



https://github.com/elastic/kibana/pull/193131/files#r1773326131